src/diffusers/loaders.py

```diff
@@ -1414,13 +1422,20 @@ def load_lora_into_text_encoder(
                 argument to `True` will raise an error.
         """
         low_cpu_mem_usage = low_cpu_mem_usage if low_cpu_mem_usage is not None else _LOW_CPU_MEM_USAGE_DEFAULT
-
         # If the serialization format is new (introduced in https://github.com/huggingface/diffusers/pull/2918),
         # then the `state_dict` keys should have `self.unet_name` and/or `self.text_encoder_name` as
         # their prefixes.
         keys = list(state_dict.keys())
         prefix = cls.text_encoder_name if prefix is None else prefix
 
+        is_peft_lora_compatible = version.parse(importlib.metadata.version("transformers")) > version.parse("4.33.1")
+
+        if not is_peft_lora_compatible:
```

Comment:
@sayakpaul @patrickvonplaten what is the approach we want to follow here? For now I am forcing users to use latest transformers.